index.js

```diff
@@ -1,29 +1,51 @@
 module.exports = {
-  extends: ['eslint-config-airbnb'].map(require.resolve),
+  env: {
+    'jest/globals': true,
+    browser: true,
+  },
+  extends: [
+    'eslint-config-airbnb',
+    'eslint-config-prettier',
+    'eslint-config-prettier/react',
+  ].map(require.resolve),
   parser: 'babel-eslint',
-  plugins: [
-    "mocha"
-  ],
+  plugins: ['jest', 'prettier'],
   rules: {
     'arrow-body-style': 0,
     'arrow-parens': ['error', 'as-needed'],
-    'camelcase': ['error'],
+    camelcase: ['error'],
+    'no-prototype-builtins': 0,
+    'import/prefer-default-export': 0,
+    'import/no-named-default': 0,
+    'import/first': 0,
     'import/no-unresolved': 0,
-    'max-len': 0,
-    'no-param-reassign': 0,
-    'no-underscore-dangle': [2, { 'allowAfterThis': true }],
-    'no-unused-vars': ['error', { vars: 'all', args: 'after-used' }],
+    'import/no-extraneous-dependencies': 0,
+    'no-underscore-dangle': [2, { allowAfterThis: true }],
+    semi: [2, 'never'],
+    'no-plusplus': ['warn'],
```

Comment:
Also, why are we using warn?

Reply:
i think i didnt want to refactor cx code in new repo :) i should of added an override in app rather than here

Reply:
> i think i didnt want to refactor cx code in new repo :)

We should get a process around this.  I would hate to see DD-web turn into doorstep with a lot of warnings and no progress.  I've never seen warnings work in practice.

Reply:
> We should get a process around this. I would hate to see DD-web turn into doorstep with a lot of warnings and no progress. I've never seen warnings work in practice.

Agreed, warnings are always ignored. If we don't want it just use error.